src/app/launcher/create-app/project-progress-createapp-nextstep/project-progress-createapp-nextstep.component.html

```diff
@@ -79,7 +151,7 @@ <h2 class="card-pf-title card-pf-title-project-progress">
   <div>
     <div class="container-fluid">
       <div class="f8launcher-continue">
-        <button class="btn btn-primary btn-lg"  (click)="launcherComponent.completed()">View New Application</button>
+        <button class="btn btn-primary btn-lg" (click)="launcherComponent.completed()">Start a new project</button>
```

Comment:
As per OSIO it button text should be "View New Application". If diff check is needed for the launcher flow then we should have check here as well `*ngIf="launcherComponent.flow === 'osio' || allCompleted !== true"`